front/src/ui/main.ts

```diff
@@ -16,16 +16,15 @@ Sentry.init({
   integrations: [
     Sentry.browserTracingIntegration({
       router,
-      tracePropagationTargets: ["localhost", /^https:\/\/app\.twinte\.net/],
     }),
     Sentry.replayIntegration({
       maskAllText: false,
     }),
   ],
+  tracePropagationTargets: ["localhost", /^https:\/\/app\.twinte\.net/],
   tracesSampleRate: 1.0,
   replaysSessionSampleRate: 0.01,
   replaysOnErrorSampleRate: 1.0,
-  logErrors: true,
```

Comment:
https://github.com/getsentry/sentry-javascript/blob/c9153578cb96ba2b317e8a1431f3119a8a56d204/docs/migration/v8-to-v9.md?plain=1#L385